frontend/src/lib/stores/hide-zero-neurons.store.ts

```
@@ -0,0 +1,12 @@
+import { StoreLocalStorageKey } from "$lib/constants/stores.constants";
+import { writableStored } from "$lib/stores/writable-stored";
+import type { Writable } from "svelte/store";
+
+export type HideZeroNeuronsMode = "show" | "hide";
+
+export type HideZeroNeuronsStore = Writable<HideZeroNeuronsMode>;
+
+export const hideZeroNeuronsStore = writableStored<HideZeroNeuronsMode>({
+  key: StoreLocalStorageKey.HideZeroNeurons,
+  defaultValue: "show",
```

I think default value should be `hide`
> Default value: hide projects with 0 neurons turned on

---

Good catch!